guides/release/services/index.md

```diff
@@ -140,3 +140,145 @@ Note `cart` being used below to get data from the cart.
 ```
 
 <!-- eof - needed for pages that end in a code block  -->
+
+### Accessing services from native classes
+
+If you want to access a service from a plain JavaScript class, you'll need to get a reference to the "[owner](https://api.emberjs.com/ember/release/modules/@ember%2Fowner)" object, which is responsible for managing services.
+
+First, we can define a class that accesses services as described above:
+
+```javascript {data-filename=app/components/cart-content/vanilla-class.js}
+import { getOwner } from '@ember/owner';
+import { service } from '@ember/service';
+
+export class VanillaClass {
+  @service shoppingCart;
+
+  someMethod() {
+    // Now you can use the service
+    this.shoppingCart.add(/* ... */);
+  }
+}
+```
+
+And then to wire up `VanillaClass` to work with `@service`, you'll need to implement a ceremony:
+
+```javascript {data-filename=app/components/cart-content/index.js}
+import { VanillaClass } from './vanilla-class';
+
+export default class CartContentsComponent extends Component {
+  @cached
+  get vanillaClass() {
+    const instance = new VanillaClass();
+
+    setOwner(instance, getOwner(this));
+
+    return instance;
+  }
+}
+```
+
+In reality, this could be any framework-construct: a service, route, controller, etc -- in this case we use a component, but this could also be done in another vanilla class that's already be wired up.
+The pattern here is to use a [`@cached`](https://api.emberjs.com/ember/5.3/functions/@glimmer%2Ftracking/cached) getter to ensure a stable reference to the class, and then using [`setOwner`]( https://api.emberjs.com/ember/5.3/functions/@ember%2Fowner/setOwner) and [`getOwner`](https://api.emberjs.com/ember/5.3/functions/@ember%2Fowner/getOwner), we finish the wiring ceremony needed to make native classes work with services.
+
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+
+Note that a stable reference in this situation means that when the property is accessed multiple times the same reference is returned. Without the `@cached` decorator, a new `VanillaClass` would be instantiated upon each access of the getter.
+
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="">
+  </div>
+</div>
+
+The exact way in which the wiring ceremony is done is up to you, but it often depends on what is needed, and community libraries may abstract away all of these if they wish.
+
+#### With arguments
+
+If your native class needs arguments, we can change the above example to instantiate the class like this:
```

Comment:
From here down seems like useful content, but maybe unrelated to services. Not sure where else it could go. Do we have a cookbook section?